README.md

```diff
@@ -39,16 +39,22 @@ TorchServe is a flexible and easy to use tool for serving PyTorch models.
 
 2. Install python pre-requisite packages
 
- - For CPU or GPU-Cuda 10.2
+ - For CPU or GPU with Cuda 11.0
```

Comment:
@harshbafna  PyTorch 1.7 also supports cuda 11.0, please add relevant steps for it as well